test/subscriber.ts

```diff
@@ -39,17 +41,10 @@ interface ClientCallback {
   (error: null|Error, client: FakeClient): void;
 }
 
-class FakePubSub {
```

Comment:
behavior changes in type refactoring makes me nervous.  Can you leave this the way it was?

Reply:
You could probably just do something like

```ts
pubsub = new FakePubSub() as unknown as PubSub;
```